doc/reference/configuration/configuration_reference.rst

```diff
@@ -3860,6 +4145,78 @@ The ``replication`` section defines configuration parameters related to :ref:`re
     | Default: 0
     | Environment variable: TT_REPLICATION_SYNC_TIMEOUT
 
+
+.. _configuration_reference_replication_synchro_queue_max_size:
+
+.. confval:: replication.synchro_queue_max_size
+
+    **Since:** :doc:`3.3.0 </release/3.3.0>`
+
+    The ``replication.synchro_queue_max_size`` option controls the maximum size of the synchronous transaction queue on a master node.
+    This parameter ensures that the queue does not grow indefinitely, potentially impacting performance and resource usage.
+
+    The size is measured in bytes, with a value of 0 disabling the limit (unlimited queue, as in earlier versions).
+
+    .. NOTE::
```

Comment:
Why note? To me this seems a part of the description itself.

Reply:
fixed